src/main/java/Tasks.java

```diff
@@ -0,0 +1,42 @@
+package packagedirectory.test;
+
+public class Tasks {
+
+    protected Message msg;
+    protected String status;
+    public static int count;
+    protected String logo;
+
+    public Tasks(Message msg) {
+        this.msg = msg;
+        this.status = "[x]";
+        this.logo = "";
+        count++;
+    }
+
+    public Message getMsg() {
+        return msg;
+    }
+
+    public void added() {
+        String output = Message.lines + "added: " + msg.getMsg() + "\n" + Message.lines;
+        System.out.println(output);
+    }
+
+    public void done() {
+        status = "[o]";
+        Message output = new Message("Nice! I've finish this task:\n" + status + " " + msg.getMsg());
+        System.out.println(output);
+    }
+
+    public void removed() {
+        Message output = new Message("Noted. I've removed this task:\n" + status + " " + msg.getMsg());
+        System.out.println(output);
+    }
+
```

Comment:
can try adding these functions ( added(), done(), removed() ) in your Message class instead of adding it in this class, so that you dont have to override it in your events, todo and deadline class. I think this will simplify your code :)